tests/__init__.py

```diff
@@ -42,7 +42,7 @@ def get_random_name(self, *args):
 class BaseTestCase(AsyncTestCase):
     @asyncio.coroutine
     def create_connection(self, cleanup=True) -> Generator[Any, None, Connection]:
-        client = yield from connect(AMQP_URL, loop=self.loop)
+        client = yield from connect(str(AMQP_URL), loop=self.loop)
```

Comment:
that's illustrated which the backward compatibility will be broken.

Reply:
in declaration of connect() url is annotaded as `str` (actually implicit `Optional[str]`), and only such argument should be provided, otherwise a change in both func spec and documentation is required, but I wouldn't recommend that

```
    :param url: `RFC3986`_ formatted broker address. When :class:`None` \
                will be used keyword arguments.
```